src/CaptureService/include/CaptureService/ClientCaptureEventCollectorBuilderImpl.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_SERVICE_CLIENT_CAPTURE_EVENT_COLLECTOR_BUILDER_IMPL_H_
+#define CAPTURE_SERVICE_CLIENT_CAPTURE_EVENT_COLLECTOR_BUILDER_IMPL_H_
+
+#include <grpcpp/grpcpp.h>
+
+#include "CaptureService/ClientCaptureEventCollectorBuilder.h"
+#include "GrpcProtos/services.grpc.pb.h"
+#include "GrpcProtos/services.pb.h"
+#include "ProducerEventProcessor/ClientCaptureEventCollector.h"
+#include "ProducerEventProcessor/GrpcClientCaptureEventCollector.h"
+
+namespace orbit_capture_service {
+
+class ClientCaptureEventCollectorBuilderImpl : public ClientCaptureEventCollectorBuilder {
```

Comment:
I would rather name it ´GrpcClientCaptureEventCollectorBuilder`, as this implementation explicitly creates the grpc version.

Reply:
Thanks for suggesting the naming. changed accordingly. Thanks for the suggestion!

I also rename the `StartStopCaptureRequestWaiterImpl` to `GrpcStartStopCaptureRequestWaiter` as it is a gPRC dependent implementation.